components/src/Modal/Modal.story.js

```diff
@@ -173,13 +136,7 @@ storiesOf("Modal", module)
     </Modal>
   ))
   .add("with select and scrolling content", () => (
-    <Modal
-      title="Edit Profile"
-      onRequestClose={() => {}}
-      primaryButton={{ label: "Submit", type: "submit", form: "myForm" }}
-      secondaryButtons={[{ label: "Cancel", onClick: () => {} }]}
-      maxWidth="456px"
-    >
+    <Modal title="Edit Profile" footerContent={modalButtons} onRequestClose={() => {}} maxWidth="456px">
```

Comment:
Should there be a story with no footerContent just so we capture that use case visually?

Reply:
Yes.